### ISSUE TITLE
Update geolocation-sensor description to mention background operations

### ISSUE THREAD

**Issue by tidoust** [w3c/web-roadmaps#334]
See use cases section:
> Only the foreground operations were possible with [GEOLOCATION-API], the background operations are completely novel.
> https://w3c.github.io/geolocation-sensor/#use-cases-categorization

We should point new background use cases out.

**Comment by prushforth** [comment]
We should also talk about how a standard for how Web sites can represent the maps on which the feature (the sensor) exists AND we should talk about how web sites can represent the geofences (also features), and whether they need visualization / styling support.  Here are some [use cases and requirements](http://maps4html.github.io/HTML-Map-Element-UseCases-Requirements/) for Web maps.

**Comment by tidoust** [comment]
@prushforth I'd say your comment goes beyond the scope of the  [Mobile roadmap](https://w3c.github.io/web-roadmaps/mobile/), which is the roadmap I had in mind when I created the issue

Now I suspect you were rather thinking about the [Spatial Data on the Web roadmap](https://w3c.github.io/web-roadmpas/sdw/)? The update would certainly be useful there as well. I created #336 for the SDW roadmap, and suggest we continue the discussion there.
